components/SelectClasses/SelectClasses.module.css

```
@@ -0,0 +1,78 @@
+.table {
+  display: block;
```

Stylelint (17.12.0)

[error] Duplicate property "display" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)